CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **BREAKING**: Removed `SagemakerHuggingfaceEmbeddingModelDriver`, use `AmazonSageMakerJumpstartEmbeddingDriver` instead.
 - **BREAKING**: Removed `SagemakerTensorflowHubEmbeddingModelDriver`, use `AmazonSageMakerJumpstartEmbeddingDriver` instead.
 - **BREAKING**: `AmazonSageMakerJumpstartPromptDriver.model` parameter, which gets passed to `SageMakerRuntime.Client.invoke_endpoint` as `EndpointName`, is now renamed to `AmazonSageMakerPromptDriver.endpoint`.
+- **BREAKING**: Removed parameter `template_generator` on `PromptSummaryEngine`.
```

Comment:
Let's call out the replacement.

Reply:
i added them as separate lines.

Reply:
Yeah but let's connect them. "Removed parameter `template_generator` on `PromptSummaryEngine` in favor of `system_template_generator` and  `user_template_generator`."